packages/web/src/pages/edit-collection-page/desktop/EditCollectionPage.tsx

```diff
@@ -47,20 +47,13 @@ export const EditCollectionPage = () => {
   useRequiresAccount()
   useIsUnauthorizedForHandleRedirect(handle)
 
-  const { data: currentUserId } = useCurrentUserId()
-  const { data: apiCollection, status } = useGetPlaylistByPermalink(
-    {
-      permalink,
-      currentUserId
-    },
-    { disabled: !currentUserId, force: true }
-  )
+  const { data: apiCollection, status } = useCollectionByPermalink(permalink)
```

Comment:
even though we're using it twice here, I slightly prefer unpacking `isError` and `isPending` here